src/pages/iconography/pictograms/design.mdx

```diff
@@ -586,7 +586,8 @@ color, see the gradient sections on IBM Design Color Usage.
     <ResourceCard
       subTitle="Color: Gradients"
       aspectRatio="2:1"
-      href="/color/#gradients">
+      href="/color/#gradients"
```

Comment:
You don't want to include a slash before the `#` when linking to a particular spot
```suggestion
      href="/color#gradients"
```